cpp/src/aztec/honk/proof_system/verifier.hpp

```diff
@@ -24,7 +24,11 @@ template <typename program_settings> class Verifier {
     Verifier& operator=(const Verifier& other) = delete;
     Verifier& operator=(Verifier&& other);
 
-    // TODO: plonk_proof is just an std::vector<uint8_t>; probably shouldn't even exist
+    // TODO(luke): plonk_proof is just an std::vector<uint8_t>; probably shouldn't even exist
```

Comment:
Not very informative is it? All over the code we have lines like this `waffle::plonk_proof proof = prover.construct_proof();`. I'd say it's both redundant and misleading (given that `std::vector<uint8>` has nothing to do with Plonk). I think this is a case where you need a good variable name not a type alias. But even if you did, why not `using plonk_proof = std::vector<uint8>` instead of the struct. To me, though, even aliasing here is a bit like doing something like this: `using vector_size = size_t`.

Reply:
Yeah, fair enough.